drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/cache/PatternInspector.java

```diff
@@ -131,6 +120,13 @@ public Conflict hasConflicts() {
     }
 
     public ActionsInspectorMultiMap getActionsInspector() {
+        final ActionsInspectorMultiMap actionsInspector = new ActionsInspectorMultiMap();
+
+        for ( final FieldInspector fieldInspector : inspectorList ) {
+            actionsInspector.addAllValues( fieldInspector.getObjectField(),
+                                           fieldInspector.getActionInspectorList() );
+        }
+
```

Comment:
Part of the problem. Even when changes to the rule happened, this blocked them from showing up in the inspections.